npm/mount-utils/src/index.ts

```diff
@@ -220,6 +39,40 @@ export function setupHooks (optionalCallback?: Function) {
   // @ts-ignore
   Cypress.on('test:before:run', () => {
     optionalCallback?.()
-    cleanupStyles()
   })
 }
+
+/**
+ * Remove any style or extra link elements from the iframe placeholder
+ * left from any previous test
+ *
+ * @ Removed as of Cypress 11.0.0
+ */
+export function cleanupStyles () {
```

Comment:
Should this also have a link to the docs?

Reply:
@rockindahizzy it looks like this does link to the v11 CT migration guide
```js
    cleanup_styles: {
      message: `\`cleanupStyles\` is no longer supported.`,
      docsUrl: 'https://on.cypress.io/migration-11-0-0-component-testing-updates',
    }
```

Reply:
Sorry I should have clarified. In the jsdoc we should add 
`* @see https://on.cypress.io/migration-11-0-0-component-testing-updates`